### PR TITLE
autofd and style

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
Updated version of https://github.com/GNOME/libglnx/pull/95

File: glnx-fdio.c [hunk 1]
```diff
@@ -204,7 +203,7 @@ open_tmpfile_core (int dfd, const char *subpath,
    * link_tmpfile() below to rename the result after writing the file
    * in full. */
 #if defined(O_TMPFILE) && !defined(DISABLE_OTMPFILE) && !defined(ENABLE_WRPSEUDO_COMPAT)
-  fd = openat (dfd, subpath, O_TMPFILE|flags, mode);
+  glnx_autofd int fd = openat (dfd, subpath, O_TMPFILE|flags, mode);
```
Comment (jlebon) [inline glnx-fdio.c:206]: Minor, but let's scope this block so that we're not relying on `fd` being shadowed later below?

**Review by jlebon — COMMENTED** [review]
One small nit, otherwise LGTM!

**Comment by gnomesysadmins** [comment]
Thank you for contributing to libglnx!

libglnx uses Bugzilla for code review.

If you have never contributed to GNOME before make sure you have read the
getting started documentation:
http://www.gnome.org/get-involved

Otherwise please visit
https://wiki.gnome.org/Newcomers
and follow the instructions there to upload your change to Bugzilla.

File: glnx-fdio.c (hunk 1 above)
Comment (cgwalters) [reply glnx-fdio.c:206]: Thanks, pushed with that change.

**Pull request closed without merging**